.changeset/wild-countries-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/vite-dev-server': minor
```

Comment:
It should be `patch` instead of `minor`.

Reply:
Ah you're right. 👍🏻